README.md

```diff
@@ -194,6 +194,7 @@ These buckets are converted later to single point/bar on the graph. Aggregate fu
 | Name | Since | Description |
 |------|:-------:|-------------|
 | `avg` | v0.8.0 | Average
+| `median` | v0.9.5 | Median
```

Comment:
Could you please remove the version number and replace it with `NEXT_VERSION` instead?

Reply:
Done and rebased against current dev.